### PR TITLE
Add "preUploadedPresentation" and "preUploadedPresentationName" to create

### LINKED ISSUE #231: Add "preUploadedPresentation" and "preUploadedPresentationName" to create
https://docs.bigbluebutton.org/3.0/development/api/#updates-to-api-in-bigbluebutton

### PR DESCRIPTION
**Pull request by nimah79**
Closes #231

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "bigbluebutton/bigbluebutton-api-php",
+  "name": "nimah79/bigbluebutton-api-php",
```
Comment (GhaziTriki) [inline composer.json:2]: Can you revert this one please?

File: src/Parameters/CreateMeetingParameters.php [hunk 2]
```diff
@@ -1449,6 +1478,8 @@ public function getHTTPQuery(): string
             'userCameraCap'                          => $this->userCameraCap,
             'meetingExpireIfNoUserJoinedInMinutes'   => $this->meetingExpireIfNoUserJoinedInMinutes,
             'meetingExpireWhenLastUserLeftInMinutes' => $this->meetingExpireWhenLastUserLeftInMinutes,
+            'preUploadedPresentation' => $this->preUploadedPresentation,
```
Comment (GhaziTriki) [inline src/Parameters/CreateMeetingParameters.php:1481]: We need to apply code formatting here.

**Review by GhaziTriki — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (nimah79) [reply composer.json]: Done.

File: src/Parameters/CreateMeetingParameters.php (hunk 2 above)
Comment (nimah79) [reply src/Parameters/CreateMeetingParameters.php]: Done.

**Review by GhaziTriki — APPROVED** [review]

**Pull request merged**